src/components/Icon/icons/index.tsx

```diff
@@ -135,6 +136,7 @@ export type IconName =
   | 'iconTrashCan'
   | 'info'
   | 'key'
+  | 'layers'
```

Comment:
rename to `iconLayers`? @chudo76 wants the icon names to match svg file names

Reply:
Sure thing!